login.html

```diff
@@ -0,0 +1,126 @@
+<body class="login-page">
```

Comment:
I don't think this is required to be saved? It should be removed and probably (git) ignore all html files unless there's a valid exception.